app/models/mixins/old_ems_mixin.rb

```diff
@@ -0,0 +1,18 @@
+module OldEmsMixin
+  extend ActiveSupport::Concern
+
+  included do
+    belongs_to :old_ext_management_system, :foreign_key => :old_ems_id, :class_name => 'ExtManagementSystem'
+  end
+
+  # Needed for metrics
+  def my_zone
```

Comment:
Just saw this now, but this is a really weird place for this method.  I get that OldEmsMixin should enhance the method, but it feels more correct for there to be a method in the base class that does `my_zone` normally, and this method can do what it does followed by calling `super`.  @agrare ?